tests/nlu/test_evaluation.py

```diff
@@ -395,37 +392,45 @@ async def test_eval_data(
         ],
     )
 
-    (_, _, persisted_path) = await rasa.nlu.train.train(
-        _config,
-        path=str(tmp_path),
-        data=data_importer,
-        component_builder=component_builder,
-        persist_nlu_training_data=True,
+    _, nlu_model_directory = rasa.model.get_model_subdirectories(
+        unpacked_trained_rasa_model
     )
-
-    interpreter = Interpreter.load(persisted_path, component_builder)
+    interpreter = Interpreter.load(nlu_model_directory, component_builder)
 
     data = await data_importer.get_nlu_data()
     (intent_results, response_selection_results, entity_results) = get_eval_data(
         interpreter, data
     )
 
     assert len(intent_results) == 46
-    assert len(response_selection_results) == 46
+    assert len(response_selection_results) == 0
     assert len(entity_results) == 46
 
 
 @pytest.mark.timeout(240)  # these can take a longer time than the default timeout
-def test_run_cv_evaluation(pretrained_embeddings_spacy_config: RasaNLUModelConfig):
+def test_run_cv_evaluation(
+    pretrained_embeddings_spacy_config: RasaNLUModelConfig
+):
     td = rasa.shared.nlu.training_data.loading.load_data(
         "data/examples/rasa/demo-rasa.json"
     )
 
+    nlu_config = RasaNLUModelConfig(
+        {
+            "language": "en",
+            "pipeline": [
+                {"name": "WhitespaceTokenizer"},
+                {"name": "CountVectorsFeaturizer"},
+                {"name": "ResponseSelector", EPOCHS: 1}
```

Comment:
Shouldn't this be `DIETClassifier`? How does that test even work without intent classifier 😆

Reply:
Can we mock the training here as well?

Reply:
Done both!